src/FixtureBuilder/ExpressionLanguage/Parser/TokenParser/Chainable/TolerantFunctionTokenParser.php

```diff
@@ -56,7 +56,7 @@ public function __construct(ChainableTokenParserInterface $functionTokenParser,
     /**
      * @inheritdoc
      */
-    public function withParser(ParserInterface $parser): self
+    public function withParser(ParserInterface $parser): parent
```

Comment:
I think this change should be reverted. The parent signature is `public function withParser(ParserInterface $parser): static` so `self` is acceptable besides being more flexible than `parent`. I didn't know you could use `parent` though so nice to learn :)